src/fa/maps.py

```diff
@@ -238,7 +238,13 @@ def __exportPreviewFromMap(mapname, positions=None):
         return previews
 
     mapname = os.path.basename(mapdir).lower()
-    mapfilename = os.path.join(mapdir, mapname.split(".")[0]+".scmap")
+    
+    # All these mapgen checks in name here and there
+    # but what can we do? :)
+    if "neroxis_map_generator" in mapname:
```

Comment:
write an `isGeneratedMap(name)` function in MapGenManager, check against regex there.